src/Error.ts

```diff
@@ -1,19 +1,11 @@
 import { BrsType, ValueKind } from "./brsTypes";
 
-let foundError = false;
-
-export function make(message: string, line: number, file?: string) {
-    foundError = true;
-    return runtime(message, line, file);
-}
-
-export function runtime(message: string, line: number, file?: string) {
-    let location = file ? `${file}: ${line}` : `Line ${line}`;
-    let error = new Error(`[${location}] ${message}`);
-    if (process.env.NODE_ENV !== "test") {
-        console.error(error.message);
+export class BrsError extends Error {
```

Comment:
Would you be open to including `lineNumber` and `fileName` as properties on BRSError instead of adding them to the message text? That way other tools can use that information to show the error in the right spot without needing to parse the message as a string, and your code that writes the errors to the console can construct the whole message based on the properties.

Reply:
Also, if you have `columnNumber` (or `beginColumnNumber` and `endColumnNumber`) available when you're creating these errors, could you add that too?